src/capabilities/serve.js

```diff
@@ -0,0 +1,51 @@
+import { capability, Schema, DID, nullable, string } from '@ucanto/validator'
+
+/**
+ * "Manage the serving of content owned by the subject Space."
+ *
+ * A Principal who may `space/content/serve/*` is permitted to perform all
+ * operations related to serving content owned by the Space, including actually
+ * serving it and recording egress charges.
+ */
+export const star = capability({
```

Comment:
**suggestion**: I've released the new version of the `w3up` capabilities package in case you want to use the new `space/content/serve/*` from that lib.

Reply:
Ooh, excelent, thanks!